src/Sylius/Behat/Behaviour/ChoosesName.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Behat\Behaviour;
```

Comment:
We always use singular form in namespaces `Sylius\Behat\Context`, `Sylius\Behat\Page` etc. so IMO we should be consistent and leave `Behaviour`